cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -408,21 +407,35 @@ public String parseLogging(Logging logging, ConfigMap externalCm) {
                 newSettings.addMapPairs(inlineLogging.getLoggers());
             }
 
-            return createPropertiesString(newSettings);
-
+            return createLog4jProperties(newSettings);
         } else if (logging instanceof ExternalLogging) {
             if (externalCm != null && externalCm.getData() != null && externalCm.getData().containsKey(getAncillaryConfigMapKeyLogConfig())) {
-                return externalCm.getData().get(getAncillaryConfigMapKeyLogConfig());
+                return addMonitorIntervalToExternalLogging(externalCm.getData().get(getAncillaryConfigMapKeyLogConfig()));
```

Comment:
my doubt is ...
This feature seems to be available only when the user uses external logging configuration (via ConfigMap) and what we are doing here is just adding a field (the `monitorInterval` one). It means that the user could do that by himself, if we document it or if the user already knows about this feature.
Or did I get it wrong on how it works?

Reply:
No, it works for internal logging as well (see method `createLog4jProperties(OrderedProperties)` overriden in EO classes). This `monitorInterval=30` is added in both cases. If user specifies his own `monitorInterval` it is respected and his value is used.